### PR TITLE
notebook to track github metrics

### PR DESCRIPTION
**Pull request by suppathak**
This PR adds a notebook which tracks github metrics from thoth-station/support repo and save them in S3 bucket.

Related:  https://github.com/open-services-group/community/issues/209

### CODE REVIEW

**Comment by suppathak** [comment]
cc @oindrillac @chauhankaranraj (I could not assign a reviewer)

File: notebooks/thoth_support_github_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,873 @@
+{
```
Comment (oindrillac) [inline notebooks/thoth_support_github_data.ipynb:1]: <p>Lets add these to the Pipfile and lock</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by oindrillac** [comment]
cc: @MichaelClifford for approval :)

**Pull request merged**